.editorconfig

```diff
@@ -1,12 +1,20 @@
 # editorconfig.org
-
 root = true
 
 [*]
 charset = utf-8
+indent_style = tab
```

Comment:
Do we really want tab based indentation? 🙂 

I don't feel strongly about shell scripts, but at least JS should be (2) space indentation maybe?

Reply:
I think we use also mostly 4 spaces with the shell script.

Reply:
Here's the reason why I propose using tabs instead of spaces: https://twitter.com/rich_harris/status/1541761871585464323

Reply:
Besides, the nice thing about tabs is that you can set your own size if you prefer bigger or shorter indentation :) But if we really prefer using spaces, I'll default it to 2 spaces, with 4 spaces for shell script and PHP, to follow PSR-2.